p2-repository/org.eclipse.collections/osgi.bnd

```diff
@@ -4,8 +4,6 @@ jcip-annotations-version=${version;===;1.0}
 
 Export-Package: \
  !about.html,!about_files, \
- *.internal*;x-internal:=true;version="${package-version}", \
- *.impl*;x-internal:=true;version="${package-version}", \
```

Comment:
@nikhilnanivadekar dropping the `impl` rule is sufficient as the following catch-all rule `*` exports everything using the correct version. However, I would still recommend exporting the `internal` packages with the` x-internal` flag set.